### PR TITLE
LPS-143183 Create PUT UserGroup for User Group Headless API

### PR DESCRIPTION
**Pull request by erickmonteiroo**
Rebased on [LPS-143108](https://issues.liferay.com/browse/LPS-143108)

### CODE REVIEW

**Comment by erickmonteiroo** [comment]
ci:test:relevant

**Comment by erickmonteiroo** [comment]
ci:test:relevant

**Comment by drewbrokke** [comment]
@erickmonteiroo got some merge conflicts. Please rebase and force-push or send a new pull.

**Comment by erickmonteiroo** [comment]
@drewbrokke rebased

File: modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserGroupResourceTest.java [hunk 1]
```diff
@@ -52,6 +52,10 @@ protected UserGroup testPostUserGroup_addUserGroup(UserGroup userGroup)
 		return _postUserGroup(userGroup);
 	}
 
+	@Override
+	protected UserGroup testPutUserGroup_addUserGroup() throws Exception {
+		return _postUserGroup();
+	}
```
Comment (drewbrokke) [inline modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserGroupResourceTest.java:58]: SF: add a line break between these methods.

**Review by drewbrokke — CHANGES_REQUESTED** [review]
Just the one SF fix needed.

**Comment by drewbrokke** [comment]
ci:test:sf

**Comment by erickmonteiroo** [comment]
ci:test:sf

**Review by drewbrokke — APPROVED** [review]

**Comment by drewbrokke** [comment]
ci:forward

**Comment by erickmonteiroo** [comment]
ci:forward

**Comment by patriciajeanperez** [comment]
ci:forward

**Comment by patriciajeanperez** [comment]
ci:stop

**Pull request closed without merging**